crates/aptos-rosetta/rosetta_cli.json

```diff
@@ -7,25 +7,24 @@
     "data_directory": "data",
     "http_timeout": 30,
     "max_retries": 5,
-    "retry_elapsed_time": 0,
-    "max_online_connections": 100,
-    "max_sync_concurrency": 64,
-    "tip_delay": 120,
-    "max_reorg_depth": 100,
+    "retry_elapsed_time": 1,
```

Comment:
Question for my understanding, where does this come from? I would assume that Coinbase would have expectations around what these values should be.